src/job-exporter/deploy/job-exporter.yaml.template

```diff
@@ -36,11 +36,11 @@ spec:
       containers:
       - image: {{ cluster_cfg["cluster"]["docker-registry"]["prefix"] }}job-exporter:{{ cluster_cfg["cluster"]["docker-registry"]["tag"] }}
         imagePullPolicy: Always
-        livenessProbe: # we found python may have deadlock and http server can not accept new conn, so use liveness probe to check if it is alive, and kill if it has deadlock. Temporary solution
```

Comment:
I think livenessProbe is a good idea. why don't we keep it as it is?

Reply:
Sure, then only livenessProbe, no readinessProbe.